Default.sublime-commands

```diff
@@ -26,14 +26,14 @@
     { "command": "quoted_printable", "args": {"encode_type": "decode"}, "caption": "Codec: Quoted-Printable decode" },
     { "command": "hex", "args": {"encode_type": "encode"}, "caption": "Codec: C-style Hex encode (ascii)" },
     { "command": "hex", "args": {"encode_type": "decode"}, "caption": "Codec: C-style Hex decode (ascii)" },
-    { "command": "idn", "args": {"encode_type": "punycode_encode"}, "caption": "Codec: Punycode encode" },
-    { "command": "idn", "args": {"encode_type": "punycode_decode"}, "caption": "Codec: Punycode decode" },
+    { "command": "idn", "args": {"encode_type": "punycode_encode"}, "caption": "Codec: RFC 3492 Punycode encode" },
```

Comment:
Could you make the string "Codec: Punycode RFC 3492 encode" and "Codec: Punycode RFC 3492 decode" instead?